### PR TITLE
[SUP-465] Upgrade GATK in dockerifle

### PR DESCRIPTION
**Pull request by als364**

### CODE REVIEW

**Comment by rtitle** [comment]
1. Can you update `CHANGELOG.md` and `config.json`? (See https://github.com/DataBiosphere/terra-docker/blob/master/CONTRIBUTING.md). Also if AoU wants to extend from this new image we would need an AoU image version bump as well.

2. Can you add a test case [here](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-gatk/tests/gatk_smoke_test.ipynb)? We should be able to have a test that imports GATK and runs a simple command, or something. These tests run as GitHub actions on PRs.

File: terra-jupyter-aou/CHANGELOG.md
Comment (Qi77Qi) [inline terra-jupyter-aou/CHANGELOG.md:5]: typo...`us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.1.8`

**Review by calbach — APPROVED** [review]

**Comment by als364** [comment]
Without test data, the most I could do is run `gatk --version`, which it looks like already exists in that notebook file.

**Comment by Qi77Qi** [comment]
[test](https://github.com/DataBiosphere/terra-docker/pull/237/checks?check_run_id=3049572282) shows gatk 4.2.0.0 is installed properly

**Review by Qi77Qi — APPROVED** [review]

**Comment by als364** [comment]
is it possible the CI hook failed because this updates two images at the same time and neither update actually exists in GCR atm?

**Comment by Qi77Qi** [comment]
@als364 images aren't pushed to gcr on PRs..a jenkins job will be kicked off once the PR is merged....test in this repo builds these images locally in each workflow

**Comment by Qi77Qi** [comment]
oh I think the test failure is becuz aou currently can't depend on latest `terra-jupyter-gatk` yet...https://github.com/DataBiosphere/terra-docker/pull/232 is the PR to bring aou image up-to-date...

**Comment by als364** [comment]
So this PR is blocked on #232?

**Comment by Qi77Qi** [comment]
yes

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**
